Updater/Client.cs

```diff
@@ -44,7 +44,6 @@ public async Task<string> StartAsync(string ipAddress, string port)
     public void Subscribe()
     {
         _communicator.Subscribe("FileTransferHandler", this);
-        // SyncUp();
     }
 
 
```

Comment:
In all the checked in files, pls make sure no error related to editorconfig are there.

Reply:
No errors are coming in my side